app/assets/javascripts/discourse/app/lib/keyboard-shortcuts.js

```diff
@@ -86,8 +86,8 @@ const DEFAULT_BINDINGS = {
   t: { postAction: "replyAsNewTopic" },
   u: { handler: "goBack", anonymous: true },
   "x r": {
-    click: "#dismiss-new,#dismiss-new-top,#dismiss-posts,#dismiss-posts-top",
```

Comment:
dismiss-posts was only mentioned here, nowhere else in our codebase. Legacy I guess?